build/victoriametrics/build.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/bash
+#
+# {{{ CDDL HEADER
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+# }}}
+
+# Copyright 2024 Guo-Rong Koh
+
+. ../../lib/build.sh
+
+PROG=victoriametrics
+VER=1.102.1
+PKG=ooce/database/victoriametrics
+SUMMARY="VictoriaMetrics"
+DESC="Fast, cost-effective monitoring solution and time series database."
+
+DATA=var/lib/victoria-metrics-data
+
+set_arch amd64
```

Comment:
While there's no requirement to get this building on omnios aarch64, we do try and make all packages fairly agnostic. Please just specify `64` bits here.
```suggestion
set_arch 64
```